server/grpc_service.go

```diff
@@ -398,14 +367,72 @@ func (s *GrpcServer) Tso(stream pdpb.PD_TsoServer) error {
 	}
 }
 
-func (s *GrpcServer) getForwardedHost(ctx, streamCtx context.Context) (forwardedHost string, err error) {
+// forwardTSO forwards the incoming TSO requests to the TSO microservice.
+func (s *GrpcServer) forwardTSO(stream pdpb.PD_TsoServer) error {
```

Comment:
Do we need to update `getGlobalTSOFromTSOServer`?

Reply:
Before the change in this pr is proved to work as expected in dev/staging, I plan to keep the other RPCs keep the current way.